planning/obstacle_velocity_planner/src/node.cpp

```diff
@@ -0,0 +1,689 @@
+// Copyright 2022 Tier IV, Inc.
```

Comment:
Tier IV -> TIER IV 
(for all files)

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/570/commits/f8742d059e4bade089a11296fd8f596ef77cd2db
DONE